website/project/warmup.rst

```diff
@@ -259,239 +247,261 @@ your code to GitHub.
-   this is just a different way in which the internal state of the game
-   changes), your “move” function should supply this information in some
-   way (otherwise, the user of your module will have to check the state
-   of the piece after every move, to see whether it has changed)
--  Since we expect to support more games in the future, you should avoid
-   having a function for moving a chess piece and a function for moving
-   a checkers piece. Otherwise, if we end up supporting 50 games, we’ll
-   end up with 50 “move” functions! (and similarly with other functions
-   in your interface)
-
-Like the previous exercise, remember that you should focus only on
-specifying the *interface* of your modules, and not their
-implementation. Make sure to review the  :ref:`warmup-deliverables` section of the
-first part of the exercise, as your updated
-design should follow the same guidelines described there.
-
-Besides the C code that specifies your design, you should also answer
-the following questions in your README file:
-
--  In general, what major changes (if any) did you have to make to your
-   design to accommodate the additional requirements?
--  In hindsight, what (if anything) would you have done differently in
-   your original design that would’ve made it easier to add support for
-   these additional requirements?
-
-Creating your shared team repository
-------------------------------------
-
-Like the previous design exercise, you will have to follow an
-*invitation URL* to create a repository. However, since the teams were
-already created in the previous design exercise, the invitation URL will
-show you all the existing teams, and all you need to do is select yours.
-
-If you join the wrong team, or mistakenly create a duplicate team,
-please notify an instructor/TA.
-
-Submission Instructions
------------------------
-
-Before submitting, make sure you’ve added, committed, and pushed all
-your code to GitHub.
-
-You will submit your code through Gradescope. Please note that only one
-team member needs to make the submission (that team member will be given
-the option to specify who else is in their team). When submitting, you
-will be given the option of manually uploading files, or of uploading a
-GitHub repository (we recommend the latter, as this ensures you are
-uploading exactly the files that are in your repository). If you upload
-your repository, make sure you select your
-``2021-design2-team-X`` repository, with “main” as the
-branch. Please note that you can submit as many times as you want before
-the deadline.
+Grading
+=======
+
+Unlike the homeworks, the grading for the design warm-up exercise is more subjective,
+as there are many valid designs one could come up with that meet the above requirements.
+Fortunately, you will have multiple opportunities to address any feedback we give you.
+Most notably, after you get your graded Design Warm-up #2 exercise back, you will be given
```

Comment:
Are we giving students feedback on Design Warm-up #1 as well or just #2?

Reply:
Yes, I've slightly reorganized the Grading section to make this clearer.